src/plugins/dashboard/server/dashboard_saved_object/migrations/migrate_to_730/migrations_730.test.ts

```diff
@@ -41,6 +41,7 @@ test('dashboard migration 7.3.0 migrates filters to query on search source', ()
         searchSourceJSON:
           '{"filter":[{"query":{"query_string":{"query":"n: 6","analyze_wildcard":true}}}],"highlightAll":true,"version":true}',
       },
+      optionsJSON: '{}',
```

Comment:
Is `optionsJSON` really defined for dashboard saved objects from pre 7.3 releases?

Reply:
Oops, yeah. Not sure about that. Better to keep it as it was. Fixed in https://github.com/elastic/kibana/pull/193067/commits/59ee48f2cb3cf54fe814652ac56e79f4265da368.